### PR TITLE
add component to count places

### PR DESCRIPTION
**Pull request by frank12421**

### CODE REVIEW

File: components/Dates/DateForm.js [hunk 1]
```diff
@@ -105,9 +133,24 @@ export default function DateForm({ locationId }) {
...
+          type="submit"
+          backgroundcolor="globalPlaceBackgroundColor"
+        >
+          Gespeichert
+        </SubmitButton>
+      )}
+      {/* <SubmitButton type="submit" backgroundcolor="globalPlantBackgroundColor">
         Speichern
-      </SubmitButton>
+      </SubmitButton> */}
```
Comment (doctype-Melvin) [inline components/Dates/DateForm.js:153]: You can remove this if it's not doing anything :)

File: styles.js [hunk 2]
```diff
@@ -8,39 +19,42 @@ export default createGlobalStyle`
...
     rgba(255, 255, 255, 0.25) 0px 1px 0px inset;
+--scriptfond: ${indie.style.fontFamily}
 }
 
 body {
     margin: 0;
-    font-family: Arial, Helvetica, sans-serif;
+    font-family: ${jost.style.fontFamily};
+
+    /* font-family: Arial, Helvetica, sans-serif; */
```
Comment (doctype-Melvin) [inline styles.js:39]: You can remove this too :)

**Review by doctype-Melvin — CHANGES_REQUESTED** [review]
Great job Frank!
I've just left a few comments about muted code. Let me know when you've removed the lines and you'll get the ✅

File: styles.js (hunk 2 above)
Comment (frank12421) [reply styles.js:39]: Thanks for the tip. I usually do that before I create a PR. I'm still working on the components, so such code snippets stay a bit in their place.

File: components/Dates/DateForm.js (hunk 1 above)
Comment (frank12421) [reply components/Dates/DateForm.js:153]: Thanks for the tip. I usually do that before I create a PR. I'm still working on the components, so such code snippets stay a bit in their place.

File: styles.js (hunk 2 above)
Comment (doctype-Melvin) [reply styles.js:39]: ✅

File: components/Dates/DateForm.js (hunk 1 above)
Comment (doctype-Melvin) [reply components/Dates/DateForm.js:153]: ✅

File: styles.js (hunk 2 above)
Comment (frank12421) [reply styles.js:39]: Sorry, I have only now realized that the Ändeurngen are still in the PR arrived. I had planned to publish this later. Thanks for the review.

**Pull request merged**